crates/plugins/mediator-coordination/src/forward/routing.rs

```diff
@@ -0,0 +1,132 @@
+use didcomm::{Message, PackEncryptedOptions, UnpackOptions};
+
+use crate::{
+    model::stateful::entity::Messages,
+    web::{error::MediationError, AppState, AppStateRepository},
+};
+/// mediator receives messages of type forward then it unpacks the messages and stores it for pickup
+/// the unpacked message is then repacked for further transmission.
+/// Note: Stored messages are not re_packed and must be before transmission in case of
+/// Rewrapping.
+pub async fn mediator_forward_process(
+    payload: &str,
+    state: &AppState,
```

Comment:
You defined a handler, but did you register it on the unified route? How will it be accessed from the server interface? Check how other DIDComm messages are handled:
https://github.com/adorsys/didcomm-mediator-rs/blob/4d19ef5bd11c644c077c19d9dd0bec3a93a3e28a/crates/plugins/mediator-coordination/src/web/handler.rs#L15-L43

Reply:
okay :+1: